thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/dashboard/ThirdEyeDashboardApplication.java

```diff
@@ -181,7 +174,6 @@ public void run(ThirdEyeDashboardConfiguration config, Environment env)
     env.jersey().register(new AutoOnboardResource(config));
     env.jersey().register(new ConfigResource(DAO_REGISTRY.getConfigDAO()));
     env.jersey().register(new CustomizedEventResource(DAO_REGISTRY.getEventDAO()));
-    env.jersey().register(new TimeSeriesResource());
```

Comment:
The above three lines are not used.
    env.jersey().register(new DataResource(anomalyFunctionFactory, alertFilterFactory));	    env.jersey().register(new DataResource(anomalyFunctionFactory, alertFilterFactory));
    env.jersey().register(new AnomaliesResource(anomalyFunctionFactory, alertFilterFactory));